File: app/src/main/java/com/song/deviceinfo/ui/codecs/CodecViewModel.java
package com.song.deviceinfo.ui.codecs;

import com.song.deviceinfo.info.MediaCodecInfo;
import com.song.deviceinfo.ui.base.BaseViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/6/3.
 */
public class CodecViewModel extends BaseViewModel {

    protected List<Pair<String, String>> getCodeCInfo() {
        return MediaCodecInfo.getCodeCInfo();
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/codecs/CodecViewModel.kt
package com.song.deviceinfo.ui.codecs

import androidx.core.util.Pair
import com.song.deviceinfo.info.MediaCodecInfo
import com.song.deviceinfo.ui.base.BaseViewModel

/**
 * Created by chensongsong on 2020/6/3.
 */
class CodecViewModel : BaseViewModel<Pair<String, String>>() {
    val codeCInfo: List<Pair<String, String>>
        get() = MediaCodecInfo.codeCInfo
}
